docs/azure_arc_jumpstart/azure_arc_k8s/aks_hybrid/aks_edge_essentials/_index.md

```diff
@@ -28,9 +30,9 @@ However, **for demo purposes only**, the below guide will allow you to use and o
 
 - In case you don't already have one, you can [Create a free Azure account](https://azure.microsoft.com/free/).
 
-- Create Azure service principal (SP).
+- Create Azure service principal (SP)
 
-    To be complete the scenario and its related automation, an Azure service principal assigned the “Contributor” role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+    To be able to complete the scenario and its related automation, Azure service principal assigned with the “Contributor” role is required. To create it, login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
```

Comment:
```suggestion
    To be able to complete the scenario and its related automation, an Azure service principal assigned with the “Contributor” role is required. To create it, login to your Azure account and run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
```